content/2021-08-19-GSoC21.rst

```diff
@@ -0,0 +1,101 @@
+#. [Hold] `WIP: ENH: improve sort_vertices_of_regions via Pythran and made it more readable <https://github.com/scipy/scipy/pull/14376>`_ : There are currently two tests we can’t pass because 1. With Pythran we can’t do inplace sort 2. The input type will change in the Pythran function
+#. [Closed] `ENH: improve _sosfilt_float via Pythran <https://github.com/scipy/scipy/pull/14473>`_  : ``_sosfilt_float`` is already implemented in Cython. We were considering to replace it but found Pythran performance is not much better than Cython's, and Pythran does not support ``object`` type, so we decided not to merge it.
+
+Since one of our goals is to find potential issues in Pythran and make Pythran better, I also bring up many issues in Pythran.
+
+**Issues**:
+
+#. [closed] `Pythran makes np.searchsorted much slower <https://github.com/serge-sans-paille/pythran/issues/1793>`_ 
+#. [closed] `Pythran may make a function slower? <https://github.com/serge-sans-paille/pythran/issues/1753>`_ 
+#. [closed] `u_values[u_sorter].searchsort would cause "Function path is chained attributes and name" but np.search would not <https://github.com/serge-sans-paille/pythran/issues/1792>`_
+#. [closed] `all_values.sort() would cause compilation error but np.sort(all_values) would not <https://github.com/serge-sans-paille/pythran/issues/1791>`_
+#. [closed] `u_values[u_sorter].searchsort would cause "Function path is chained attributes and name" but np.searchsort would not <https://github.com/serge-sans-paille/pythran/issues/1792>`_
+#. [closed] `Support scipy.special.binom? <https://github.com/serge-sans-paille/pythran/issues/1804>`_
+#. [closed] `Got AttributeError: module 'scipy' has no attribute 'special' when building scipy with special import <https://github.com/serge-sans-paille/pythran/issues/1815>`_
+#. [closed] `Got compilation error when the inner variable type changes <https://github.com/serge-sans-paille/pythran/issues/1818>`_
+#. [closed] `Can't index an 2d array like a1[int, tuple] <https://github.com/serge-sans-paille/pythran/issues/1819>`_
+#. [closed] `keep_dims is not supported in np.mean() <https://github.com/serge-sans-paille/pythran/issues/1820>`_
+#. [closed] `can't use np.expand_dims with specified keyword argument <https://github.com/serge-sans-paille/pythran/issues/1850>`_
+#. [open ] `bus error on Mac but works fine on Linux for _count_paths_outside_method pythran version <https://github.com/scipy/scipy/issues/14315>`_ 
+#. [open ] `array assignment res[cond1] = ax[cond1] works fine for int[] or float[] or float[:,:] but not int[:,:] <https://github.com/serge-sans-paille/pythran/issues/1858>`_
+
+Work Left
+=========
+
+As the project proceeded, I found it was difficult to find 
+suitable algorithms to be implemented. Moreover, in our past experience 
+with Pythran,  we often run into some things that are easy to get wrong. 
```

Comment:
It'd be nice to add what those things were. E.g.: `... to get wrong, such as the use of integer dtypes in a portable way, and using arrays that are views as input to a Pythran-ized function".